examples/Demo/FluentUI.Demo.Client/Documentation/Components/Label/Examples/LabelDisabled.razor

```diff
@@ -0,0 +1,3 @@
+﻿<div style="display: flex; gap: 12px; margin-bottom: 24px;">
```

Comment:
This is a comment I've had: we're using ‘too many’ styles to get the right result.
I think we need to use examples with only the FluentUI Blazor components.

If something is missing to get the expected result, it's because we've forgotten something: property, other component, etc.

What is your opinion?

Reply:
Yes, we need to make this more 'simple'